x-pack/plugins/case/server/routes/api/cases/status/get_status.test.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { kibanaResponseFactory, RequestHandler } from 'src/core/server';
+import { httpServerMock } from 'src/core/server/mocks';
+
+import {
+  createMockSavedObjectsRepository,
+  createRoute,
+  createRouteContext,
+  mockCases,
+} from '../../__fixtures__';
+import { initGetCasesStatusApi } from './get_status';
+import { CASE_STATUS_URL } from '../../../../../common/constants';
+
+describe('GET status', () => {
+  let routeHandler: RequestHandler<any, any, any>;
+  beforeAll(async () => {
+    routeHandler = await createRoute(initGetCasesStatusApi, 'get');
+  });
+
+  it(`returns the status`, async () => {
+    const request = httpServerMock.createKibanaRequest({
+      path: CASE_STATUS_URL,
+      method: 'get',
+    });
+
+    const theContext = await createRouteContext(
+      createMockSavedObjectsRepository({
+        caseSavedObject: mockCases,
+      })
+    );
+
+    const response = await routeHandler(theContext, request, kibanaResponseFactory);
+    expect(theContext.core.savedObjects.client.find).toHaveBeenNthCalledWith(1, {
+      fields: [],
```

Comment:
nit to make a `findArgs` variable that has the fields that remain constant through these tests and spread it throughout the tests. ex:
```
const findArgs = {
      fields: [],
      page: 1,
      perPage: 1,
      type: 'cases',
}
```